terraform/aks/databases.tf

```diff
@@ -1,3 +1,23 @@
+module "redis-cache" {
+  source = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/redis?ref=testing"
+
+  namespace                 = var.namespace
+  environment               = local.environment
+  azure_resource_prefix     = var.azure_resource_prefix
+  service_name              = local.service_name
+  service_short             = var.service_short
+  config_short              = var.config_short
+  azure_capacity            = var.redis_cache_capacity
+  azure_family              = var.redis_cache_family
+  azure_sku_name            = var.redis_cache_sku_name
+  name                      = "cache"
```

Comment:
could you add:
```
  patch_schedule {
    day_of_week    = "Sunday"
    start_hour_utc = 01
  }
 ```
 Just an extra setting to ensure if maintenance was to happen it happens outside hours?